### ISSUE TITLE
Silent crash on start when morphs.ini refers to mod that isn't loaded.

### ISSUE THREAD

**Issue by mrudat** [napmouse/autoBodyAE#29]
Not sure if it's a bug as such, but as far as I can tell, if your morphs.ini points (at individual NPCs?) from mods that aren't loaded (say, if you import [someone else's configuration](https://www.nexusmods.com/skyrimspecialedition/mods/62399)), autobody silently crashes on startup; disabling autobody or the incorrect configuration file stops it from crashing, but obviously, that's not the desired state.

To fix the issue, I commented out the sections of the configuration file referring to mods that I didn't have.

It would be nice if this didn't cause autobody to crash, or at least ended up with an error in the log file pointing to why it crashed.

**Comment by napmouse** [comment]
Thanks for the report, I never caught this. I've been busy lately but I'll fix it for the next version when I can.

**Comment by napmouse** [comment]
So I looked into this, and it turns out its already a handled error case and I just forgot. It's not silent, and it doesn't crash the mod. Here's the log when I tested this case by just chopping off the bottom part of that config: 

>[error] SMSkyrim.esp|E0DF5A is not a valid key!
[trace] SMSkyrim.esp is the owning mod.
[trace] E97C is the character ID
[error] SMSkyrim.esp|E97C is not a valid key!
[trace] Yuri The Elder Trainer.esp is the owning mod.
[trace] D62 is the character ID
[error] Yuri The Elder Trainer.esp|D62 is not a valid key!

However, the entire config & morph setup *does* cause crashing, so there's clearly something wrong. But it's not this.
